[DOCS] Documentation Addition (Readme updates) (#2495)
ROCm/composable_kernel: ACRONYMS.md

# Acronyms in Composable Kernel

The following acronyms are used in the Composable Kernel codebase:

| Acronym | Expansion | Explanation |
|---------|-----------|-------------|
| B2B     | Back-to-Back | Fused or sequential kernel operations (e.g., B2B GEMM) |
| BF16    | Brain Floating Point 16 | 1 Signed bit, 8 Exponent bits, 7 Mantissa bits |

> | BF8     | 8-bit Brain Floating Point | 1 Signed bit, 3 Exponent bits, 4 Mantissa bits |

Reviewer: Thanks! Nit, 'Sign', not 'Signed'. Also checked wiki and IEEE calls 'Significand' what I was calling 'Mantissa' so that should be also changed

# Acronyms in Composable Kernel

> This file lists technical acronyms found throughout the Composable Kernel codebase.

Reviewer 2: ```suggestion
The following acronyms are used in the Composable Kernel codebase:
```

| Acronym | Expansion | Explanation |
|---------|-----------|-------------|

> | API     | Application Programming Interface | Interface for interacting with software components |

Reviewer 3: API seems too obvious.

> | B2B     | Back-to-Back | Fused or sequential kernel operations (e.g., B2B GEMM) |

Reviewer 2: Does it described a fused op or a sequential op, or are fused and sequential ops the same thing?

> | BF16    | Brain Floating Point 16 | 16-bit floating point format for ML workloads |

Reviewer 2: I would personally put in that it's equivalent to a float, but I don't know if that's more detail than you want here.

Reviewer: "equivalent to float" is not precise. 1 sign bit, 8 exponent bits, 7 mantissa bits. E8M7 is another notation for that

Reviewer 3: I changed the description, please have a relook @tenpercent

> | BF8     | 8-bit Brain Floating Point | 8-bit floating point format, variant for ML |

Reviewer 2: See BF16

| BLAS    | Basic Linear Algebra Subprograms | Standardized library for linear algebra routines |

> | C++     | C Plus Plus | Programming language used for CK |

Reviewer 2: I probably wouldn't put this in :)

> | CK      | Composable Kernel | The main project/library name |

Reviewer 2: ```suggestion
| CK      | Composable Kernel | The shortened name of the Composable Kernel library. |
```

> | CPU     | Central Processing Unit | General-purpose processor |

Reviewer 2: I'd mention that this is equivalent to HOST.

| CUDA    | Compute Unified Device Architecture | NVIDIA's GPU programming platform |
| DLA     | Deep Learning Accelerator | Specialized hardware for deep learning workloads |

> | DRAM    | Dynamic Random-Access Memory | Main memory (global memory on GPU) |

Reviewer 2: ```suggestion
| DRAM    | Dynamic Random-Access Memory | Main memory. Global memory on GPU |
```
